server/src/routes/api/v1/boardGamesRouter.js

```diff
@@ -44,4 +45,17 @@ boardGamesRouter.get("/:id", async (req, res) =>{
 	}
 })
 
+boardGamesRouter.delete("/:id", async (req, res) =>{
+	const { id } = req.params
+	try {
+		const rowsDeleted = await BoardGame.query().deleteById(id)
+		if (rowsDeleted === 1) {
+			return res.status(204).send()
```

Comment:
we should send something back as json - even just a message that the delete action was successful to indicate to other developers